cli/spec/kontena/cli/stacks/yaml/reader_spec.rb

```diff
@@ -312,7 +310,12 @@
 
     it 'expands build option to absolute path' do
       outcome = subject.execute
-      expect(outcome['services']['webapp']['build']['context']).to eq(fixture_path(''))
```

Comment:
It's a bit of a mystery how this test ever passed after the #2707 (which turned services into an array)